### PR TITLE
Post for eResearch Australasia workshop and BoF

### PR DESCRIPTION
**Pull request by rowlandm**
Please review

### CODE REVIEW

**Comment by rowlandm** [comment]
I'll approve Monday lunchtime by default.

File: _posts/2018-11-03-eResearch-Australasia-workshop-and-BoF.markdown [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+
+What was great about both meetings was that we had a range of people from all over Australian and New Zealand all willing to help grow the community.
+
+If you would like to help out, there are a number of things you can do:
+ 
+- Sign up for the [google groups email list](<https://groups.google.com/d/forum/rse-nz-au>)
+- Join our monthly meeting of volunteers by letting us know via the Google Group.
+- Comment on our open github issue of the summary of the BoF and workshop: [https://github.com/rse-aunz/rse-au/issues/49](<https://github.com/rse-aunz/rse-au/issues/49>)
+- Comment on other optics on the github issues folder: [https://github.com/rse-aunz/rse-au/issues](<https://github.com/rse-aunz/rse-au/issues>)
```
Comment (manodeep) [inline _posts/2018-11-03-eResearch-Australasia-workshop-and-BoF.markdown]: Although I am not sure I understand what is meant by "other optics"

**Comment by rowlandm** [comment]
should have been topics...

**Comment by rowlandm** [comment]
I think its ok as the workshop was reviewed via thos BoF slides and the
bulk of the text was sent out for review last week as part of aero
submission.

Also there were emails sent by Heidi and myself based on that same
submission.

Happy to discuss,  just worried everyone is too busy to review and the is a
need to get it out for extra emails ....

**Comment by manodeep** [comment]
@rowlandm Ahh okay :)

**Pull request merged**